### PR TITLE
Add explicit whitelist of RPC commands for luigid

### PR DESCRIPTION
**Pull request by bwall**
Limit calls on Scheduler from API to only those required for functionality.

Basic functionality testing was conducted, but not extensive testing.

### CODE REVIEW

**Comment by erikbern** [comment]
nice!

File: luigi/server.py [hunk 1]
```diff
@@ -68,6 +68,26 @@ def initialize(self, scheduler):
...
+            'add_task',
+            'add_worker',
+            'dep_graph',
+            'disable_worker',
+            'fetch_error',
+            'get_work',
+            'graph',
+            'inverse_dep_graph',
+            'ping',
+            'prune',
```
Comment (Tarrasch) [inline luigi/server.py]: Not sure if this actually should be public. But whatever.

**Comment by Tarrasch** [comment]
Nice! Thanks :)

File: luigi/server.py (hunk 1 above)
Comment (Tarrasch) [inline luigi/server.py]: To clarify. I'm not sure it makes sense for clients to be able to call `prune`. Many other of the API calls does call `prune` internally themselves, like `graph`, `inverse_dep_graph` etc.

This was just a minor comment so I just merged right away.

**Pull request merged**